src/plugins/internal/InternalInstallPlugin.ml

```diff
@@ -146,9 +77,12 @@ let install pkg argv =
       fun fn -> fn
   in
 
-  let install_file ?tgt_fn src_file envdir =
+  let install_file ?(with_destdir=true) ?tgt_fn src_file envdir =
```

Comment:
Also, note that it removes a hack for windows, which I think is a rather positive point.